src/coreComponents/constitutive/solid/porosity/BiotPorosity.hpp

```diff
@@ -144,19 +144,17 @@ class BiotPorosityUpdates : public PorosityBaseUpdates
   virtual void updateFromPressureAndTemperature( localIndex const k,
                                                  localIndex const q,
                                                  real64 const & pressure, // current
-                                                 real64 const & pressure_k, // last iteration (for sequential)
+                                                 real64 const & GEOS_UNUSED_PARAM( pressure_k ), // last iteration (for sequential)
```

Comment:
need to check for temperature, has to be somewhat symmetrical to pressure treatment

Reply:
@castelletto1 Can you please have a look if the temperature treatment is consistent? The porosity update formulation is changed from effective to total stress increment.